Libplanet.Tools/download.mjs

```diff
@@ -119,7 +143,26 @@ Report this issue from https://github.com/planetarium/libplanet/issues
 ${err}
 -------------------------------------------------------------------------------
     `);
-    throw e;
+    throw err;
+  }
+  if (platform === "darwin" && arch === "arm64")
```

Comment:
Wouldn't it better to ad-hoc sign the executable on Intel Macs too?

Reply:
makes sense, but I didn't have Intel Mac to test😅 .

Reply:
Okay, never mind then.  Anyway, could you resolve the conflict?